data/liveinst/liveinst

```diff
@@ -26,6 +26,11 @@ if [ "$(id -u)" -ne 0 ]; then
     exec pkexec "$0" "$@"
 fi
 
+# pkexec clears DBUS_SESSION_BUS_ADDRESS from environment
+if [ -z "$DBUS_SESSION_BUS_ADDRESS" ]; then
+    export DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/${PKEXEC_UID}/bus
+fi
```

Comment:
I wonder how many of these issues could impact us. Anyway, let's go one by one :)